src/lib/core/components/visualizations/implementations/HistogramVisualization.tsx

```diff
@@ -444,17 +428,16 @@ export function histogramResponseToData(
 function getRequestParams(
   studyId: string,
   filters: Filter[],
-  entity: StudyEntity,
-  variable: HistogramVariable,
-  overlayEntity?: StudyEntity,
-  overlayVariable?: StudyVariable, // TO DO: ?CategoricalVariable?
+  variable: Variable,
+  variableType: 'number' | 'date',
+  overlayVariable?: Variable,
```

Comment:
Well this is certainly much cleaner - not passing variable and entity for both variables!